src/components/pages/career-details/career-details.stories.mdx

```diff
@@ -8,27 +7,6 @@ import { Form } from './new-career-form/career-form';
 
 Here the parts to build the career details page.
 
-## Job Description
```

Comment:
Do you use the stories? I'm happy to remove them if nobody uses them. It's maybe a little bit over the edge for us and was good in the beginning as a playground. But whenever I see stories these days it feels like outdated, unused, obsolete.